### DISCUSSION TITLE
[2.0.0] 스위프트 패키지 리소스의 이미지 Unassigned 이슈

### DISCUSSION THREAD

**Discussion by x-0o0** [ku-ring/app-ios-v2#100] [iOS 개발]
## 이슈 설명
레포를 clone 하면 스위프트 패키지 리소스의 모든 이미지들이 unassigned 라는 경고가 뜸

## 재현방법
1. 현재 레포(ku-ring/ios-app) clone
2. Scheme 을 KuringModulePackage 로 설정
3. 빌드
4. Warnings 확인

## 기대결과
에러 및 워닝이 없어야 함

## 스크린샷
![image](https://github.com/ku-ring/ios-app/assets/53814741/1a904abb-2687-4f76-bf00-1c4908e12861)

## 버전 정보

| 플랫폼 | 버전 |
| --- | --- |
| 쿠링 앱 | 2.0 |
| iOS | 17.0 |
| Xcode | 15.0 |

**Comment by x-0o0** [comment]
스위프트 패키지 매니저의 버그로 추정
추정되는 버그에 맞게 해결 시도 후 해결 되면 [스위프트 패키지 매니저](https://github.com/apple/swift-package-manager)에 이슈 등록

**Comment by x-0o0** [comment]
[스위프트 패키지 매니저 | apple](https://github.com/apple/swift-package-manager) 에 이슈 등록
https://github.com/apple/swift-package-manager/issues/6981

**Comment by lgvv** [comment]
> [스위프트 패키지 매니저 | apple](https://github.com/apple/swift-package-manager) 에 이슈 등록 [apple/swift-package-manager#6981](https://github.com/apple/swift-package-manager/issues/6981)

To apple...

**Comment by x-0o0** [comment]
[애플 답변](https://github.com/apple/swift-package-manager/issues/6981#issuecomment-1753280780)에 따라 피드백 지원 쪽에 피드백 남김
<img width="1321" alt="스크린샷 2023-10-10 오후 6 59 32" src="https://github.com/ku-ring/ios-app/assets/53814741/5488baed-b921-42af-a09e-cd9062f24162">
